app/src/main/java/org/wikipedia/dataclient/wikidata/Claims.java

```diff
@@ -0,0 +1,63 @@
+package org.wikipedia.dataclient.wikidata;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.google.gson.annotations.SerializedName;
+
+import org.apache.commons.lang3.StringUtils;
+import org.wikipedia.dataclient.mwapi.MwResponse;
+import org.wikipedia.json.PostProcessingTypeAdapter;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+@SuppressWarnings("unused")
+public class Claims extends MwResponse implements PostProcessingTypeAdapter.PostProcessable {
+    @Nullable private Map<String, List<Claim>> claims;
```

Comment:
Don't need to implement `PostProcessable` here.

Reply:
Done